src/components/Common/Sidebar/ZekSidebar.vue

```diff
@@ -294,6 +293,9 @@ export default {
           })
         }
       })
+    },
+    expandedStatus(index){
+      this.$emit('expandedStatus', index)
```

Comment:
Function and event name too vague for the functionality, it could be better if named `onToggle` or `onTitleClick` or `onExpand`